src/LinqToTwitter5/LinqToTwitter.Shared/DirectMessageEvents/TwitterContextDirectMessageEventsCommands.cs

```diff
@@ -144,7 +144,7 @@ public async Task<DirectMessageEvents> RequestButtonChoiceAsync(ulong recipientI
         /// <param name="callToActions">List of Call to Action, which creates buttons in the message.</param>
         /// <param name="cancelToken">Async cancellation token.</param>
         /// <returns>Direct message events data.</returns>
-        async Task<DirectMessageEvents> NewDirectMessageEventAsync(ulong recipientID, string text, Attachment attachment = null, QuickReply quickReply = null, IEnumerable<CallToAction> callToActions = null, CancellationToken cancelToken = default(CancellationToken))
+        public async Task<DirectMessageEvents> NewDirectMessageEventAsync(ulong recipientID, string text, Attachment attachment = null, QuickReply quickReply = null, IEnumerable<CallToAction> callToActions = null, CancellationToken cancelToken = default(CancellationToken))
```

Comment:
This was a breaking change that I found after compiling the entire project, including client code in the ConsoleSamples project. It makes calls to other `NewDirectMessageEventAsync` overloads ambiguous because of the default parameters. I pushed an update that removes defaults from parameters and makes other overloads call this one explicitly.

Reply:
Thanks!